src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -228,13 +228,8 @@ def ZCLReadAttribute(self, cluster, attribute, nodeid, endpoint, groupid, blocki
         if res != 0:
             raise self._ChipStack.ErrorToException(res)
 
-        commandSenderHandle = self._dmLib.pychip_GetCommandSenderHandle(device)
-        im.ClearCommandStatus(commandSenderHandle)
-        res = self._Cluster.ReadAttribute(
-            device, cluster, attribute, endpoint, groupid, commandSenderHandle != 0)
-        if blocking:
-            # We only send 1 command by this function, so index is always 0
-            return im.WaitCommandIndexStatus(commandSenderHandle, 1)
+        self._Cluster.ReadAttribute(
```

Comment:
Will it wait for the response? Why is it different for commands and attributes?

Reply:
We only enabled im for commands, so it will mistakenly wait for a callback that will never be called currently.

Reply:
I see, thanks.